docs/competition/teams/role-descriptions/kit-return.md

```diff
@@ -19,35 +19,42 @@ rejected. When presented with a RUB full of kit, kit return staff should:
 
 **Don't let people leave the competition for good with any kit if they aren't on the list of teams authorised to keep their kit**
```

Comment:
Could we add a list of required equipment and paperwork to this document?

Reply:
This already exists at the bottom of the page.